src/test/resources/api/azure/users/_delete/rs.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
remove that if you have nothing to validate. status validation will be failry enough